app/services/forest_liana/oidc_client_manager.rb

```diff
@@ -15,7 +15,7 @@ def self.get_client_for_callback_url(callback_url)
               registration_endpoint: configuration['registration_endpoint']
             })
           else
-            client_credentials['client_id'] = ForestLiana.forest_client_id
+            client_credentials = { 'client_id' => ForestLiana.forest_client_id }
```

Comment:
I guess it fixed the case with a defined `forest_client_id`, right?

Reply:
Yes !